lib/src/rules/avoid_returning_null_for_future.dart

```diff
@@ -63,17 +63,17 @@ class _Visitor extends SimpleAstVisitor<void> {
     if (parent == null) return;
 
     DartType? returnType;
-    bool? isAsync;
+    bool isAsync;
     if (parent is FunctionExpression) {
       returnType = parent.declaredElement?.returnType;
-      isAsync = parent.body?.isAsynchronous;
+      isAsync = parent.body.isAsynchronous;
```

Comment:
With the change to `declaredElement` you could get `isAsynchronous` from `ExecutableElement`.